### ISSUE TITLE
Negotiate API for Hydrology Layer

### ISSUE THREAD

**Issue by hatchla** [ElektraInitiative/PermaplanT#1275]
### Tasks

 - [ ] In Backend
 - [ ] In Frontend

**Comment by markus2330** [comment]
What is "Negotiate API"? Do you mean designing the endpoints?

**Comment by hatchla** [comment]
> What is "Negotiate API"? Do you mean designing the endpoints?

Yes, Yannis and I need to do that together.

**Comment by markus2330** [comment]
Ok! As said, improvements are very welcome but the endpoints should be the same for all the layers (for overlapping functionality).

**Comment by horenso** [comment]
@danielsteinkogler You are working on a mechanism for creating layers, is this for layers of any type?

**Comment by markus2330** [comment]
No, for now only layers of type "additional layers" (see usecase doc/usecases/current/additional_layer.md) can be created. The one instance of them we currently have is the "drawing layer".
